response_operations_ui/templates/collection_exercise/view-sample-ci-summary.html

```diff
@@ -12,16 +12,16 @@
 }%}
 {% set formTypeCIVersionTableDataRows = [] %}
 
-{% for collection_instrument in collection_instruments %}
-    {% set form_type = collection_instrument["classifiers"]["form_type"] %}
+{% for ci_registry_instrument in ci_registry_instruments %}
+    {% set form_type = ci_registry_instrument["classifier_value"] %}
     {% do formTypeCIVersionTableDataRows.append(
         {
             "tds": [
                 {
                     "value": form_type
                 },
                 {
-                  "value": "Nothing selected",
+                  "value": ci_registry_instrument["ci_version"],
                   "tdClasses": "ons-u-ta-centre"
                 }, 
                 {
```

Comment:
In the below code, shouldn't the button text change to 'Edit version' if the version has been selected?

Reply:
You're right I didn't see the And in the acceptance criteria